Makefile

```diff
@@ -13,6 +13,10 @@ lint lint-fix install build test-unit test-integration deploy clean clean-deploy
 			echo "> Module $${module}/$(CLOUD)"; \
 			$(MAKE) -C modules/$${module}/$(CLOUD) $@ || exit 1; \
 		done; \
+	elif [ "$(CLOUD)" = "databricks" ]; then \
+		`node scripts/modulesort.js`; \
+		echo "> going with databricks ${ROOT_DIR}"; \
```

Comment:
This comment could be more general and you can pass the parameter $(CLOUD) in case we reuse the structure for another clouds

Reply:
That's right, good catch. Actually I didn't even need to call the modulesort script